src/components/HomeSearchSection/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { useRouter } from 'next/router'
+
+export default function HomeSearchSection() {
+  const router = useRouter();
+
+  const handleSearchClick = () => {
+    router.push("/product-list");
```

Comment:
Please use `'` for in typescript code.
```suggestion
    router.push('/product-list');
```